src/core/cfg.cpp

```diff
@@ -1932,6 +1932,11 @@ class CFGVisitor : public ASTVisitor {
         return true;
     }
 
+    bool visit_ellipsis(AST_Ellipsis* node) override {
+        // TODO:
+        return true;
+    }
```

Comment:
Or you could just not add it there and then we will abort when we try to execute a function containing it, which sounds like it is good enough for now.  But I think we don't need this visit function.